lib/school.rb

```diff
@@ -0,0 +1,53 @@
+class School
+  attr_reader :start_time, :hours_in_school_day, :student_names
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+    
+  end
+
+  def add_student_name(name)
+    @student_names << name
+  end
+
+  def end_time
+    int_start_time = @start_time.to_i
+    endtime = "#{(int_start_time+hours_in_school_day)}:00"
+  end
+
+  def is_full_time?
```

Comment:
This is correct, but you can probably ditch the `if else` statement and instead have

`@hours_in_school_day > 4`